library/std/src/thread/tests.rs

```diff
@@ -316,3 +316,9 @@ fn test_scoped_threads_drop_result_before_join() {
     });
     assert!(actually_finished.load(Ordering::Relaxed));
 }
+
+fn _test_scoped_threads_nll(x: &u8) {
```

Comment:
```suggestion
#[test]
fn test_scoped_threads_nll(x: &u8) {
```

Reply:
I believe this change would produce a warning for the function being unused. We're only testing whether the thing compiles here, not actually running it.

Reply:
Ah, wait, you also added `#[test]`. Those don't support function arguments though.

Reply:
Ah, right.

Could you nest it within a `#[test]` function that calls it? I think there's value in actually running this to see if it smokes anything out. We have other similar tests that do compile and run.